mono/metadata/handle.h

```diff
@@ -341,12 +341,17 @@ void mono_handle_verify (MonoRawHandle handle);
 #define MONO_HANDLE_PAYLOAD_OFFSET_(PayloadType) MONO_STRUCT_OFFSET(PayloadType, __raw)
 #define MONO_HANDLE_PAYLOAD_OFFSET(TYPE) MONO_HANDLE_PAYLOAD_OFFSET_(TYPED_HANDLE_PAYLOAD_NAME (TYPE))
 
-#define MONO_HANDLE_INIT ((void*) mono_null_value_handle)
+// unused
+//#define MONO_HANDLE_INIT ((void*) mono_null_value_handle)
```

Comment:
If unused, then please remove.

Reply:
ok

Reply:
Give a chance for author to protest that really it might be useful, just isn't used currently, and here is a hypothetical example...? My removal is more "empirical" than based on full understanding -- it just happens to be unused currently. But you know better (whether or not you are the original author).